### ISSUE TITLE
css-plugin-base works different in 0.19.39 and  0.19.40 but not OK in either?

### ISSUE THREAD

**Issue by jonaskello** [systemjs/systemjs#1480]
I'm trying to build a loader for modular css by building on `css-plugin-base` in `plugin-css`. To start I just did the simplest thing I could think of and returned the loaded style and a hard-coded module source.

If I do this in systemjs@0.19.40 (jspm@0.17.0-beta.31) then the modulesource gets loaded but the style does not get injected into head.

If I do this in systemjs@0.19.39 (jspm@0.17.0-beta.29) then the modulesource does not get loaded but the style gets injected into head.

I would think that both things should happen, both loading of modulesource and injecting of style?

Here is my simple example code:

cssm.js
```js
var CSSPluginBase = require('css/css-plugin-base.js');

module.exports = new CSSPluginBase(function compile(style, address, opts) {

    var loader = this;

    var moduleSource = "export default { a: 'b' };";
    var moduleFormat = "esm";

    return {
        css: style,
        map: null,

        // style plugins can optionally return a modular module
        // source as well as the stylesheet above
        moduleSource: moduleSource,
        moduleFormat: moduleFormat
    };

});
```

src/app.js
```js
import styles from './test.cssm!';

console.log("styles", styles);

document.querySelector('main#outlet').innerHTML = `
  <div class="olle">A0 style</div>
  <div class="test1 test2">B0 style</div>
`;
```

src/test.cssm
```
.test1 {
    font-weight: bold;
}

.test2 {
    color: red;
}
```

**Comment by jonaskello** [comment]
In both cases I am using plugin-css@0.1.32.

**Comment by guybedford** [comment]
@jonaskello thanks, this was a regression in SystemJS - should be fixed in https://github.com/systemjs/systemjs/commit/cfcc20154c409bbd94f2d94633220c1e1d366f81 and https://github.com/systemjs/systemjs/commit/3e2246a62b7ceea92be95b60168d3d8f516a3b19.

**Comment by guybedford** [comment]
Released in 0.19.41.
